### PR TITLE
Cockpit 231

### PR DESCRIPTION
**Pull request by marusak**
Preview: https://marusak.dev/cockpit-project.github.io/blog/cockpit-231.html

### CODE REVIEW

**Comment by marusak** [comment]
Updated also URLs, this is ready for review.

File: _posts/2020-10-29-cockpit-231.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly. Here are the release notes from Cockpit version 231.
+
+### Replace system's graph page with a completely new USE method page
+
+System graphs (reachable after clicking on 'View graphs' on the overview page) were replaced with
+a dedicated page for the analyzing current and the historical performance of a machine.
+This page (reachable after clicking on 'View details and history' on the overview page)
+shows current metrics as well as historical events based on the
+[USE](http://www.brendangregg.com/usemethod.html) method.
```
Comment (martinpitt) [inline _posts/2020-10-29-cockpit-231.md:20]: I don't think we need to be extra verbose about the old page, it just looks like this is being written twice. Also fixing some grammar issues.

```suggestion
The system graphs were replaced with a dedicated page for analyzing the current and historical performance of a machine. This page (reachable after clicking on 'View details and history' on the Overview page) shows current metrics as well as historical events based on the [USE](http://www.brendangregg.com/usemethod.html) method.
```

File: _posts/2020-10-29-cockpit-231.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+### Machines: Reimplement the design of the main VMs list
+
+Details of virtual machines are now displayed on a separate page that contains all information in one
+place. There is no need anymore to change between tabs to see different properties of a virtual machine.
+
+![VM subpage](/images/vm-details.png){:.screenshot}
+
+### Logging of remote IP addresses
+
+Cockpit now includes the remote IP address when logging user logins (and login attempts). This means that the IP address will appear in the system logs, in the output of tools such as w or who, and in the login-time messages about when the last successful login was, and any failed login attempts. It also means that PAM modules can now filter logins based on IP address (for example, temporarily blocking a subnet when there are too many failed login attempts).
```
Comment (martinpitt) [inline _posts/2020-10-29-cockpit-231.md:33]: We had this feature before, it was dropped with the introduction of cockpit-tls.

```suggestion
Cockpit records the remote IP address for user logins (and failed attempts) again. The IP address will appear in the system logs, in the output of tools such as `w` or `who`, and in the login-time messages about when the last successful login happened, and any failed login attempts. It also means that PAM modules can filter logins based on IP address (for example, temporarily blocking a subnet when there are too many failed login attempts).
```

**Review by martinpitt — CHANGES_REQUESTED** [review]
Dakujem!

**Review by martinpitt — APPROVED** [review]

**Review by allisonkarlitskaya — APPROVED** [review]

**Pull request merged**